src/battle.ts

```diff
@@ -3544,6 +3556,17 @@ export class Battle {
 			} else if (args[3] === 'item') {
 				pokemon.item = '(exists)';
 			}
+			if (this.rules['Force Open Team Sheets'] && kwArgs.name) {
+				pokemon.name = kwArgs.name;
+				if (kwArgs.item) pokemon.item = Dex.items.get(Dex.sanitizeName(kwArgs.item)).name;
```

Comment:
You may use either ``.get().name`` or ``sanitizeName()``, but not both. That's double escaping.